starters/features/drizzle/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "description": "Headless TypeScript ORM with a head 🐲",
+  "__qwik__": {
+    "displayName": "Integration: Drizzle (Database ORM)",
+    "priority": -10,
+    "viteConfig": {},
+    "docs": [
+      "https://qwik.builder.io/integrations/integration/drizzle/",
+      "https://orm.drizzle.team/docs/overview"
+    ],
+    "nextSteps": {
+      "title": "Next Steps",
+      "lines": [
+        " Drizzle was installed with a simple DB schema and some demo routes,",
+        "",
+        " Neon serverless Postgres was configured by default,",
+        " but you can change to whatever database you wish,",
+        "",
+        "",
+        " After adding your database credentials to the .env file,",
+        " run `npm run drizzle:migrate` to push the schema to the database.",
+        " Drizzle studio was also added which you can access by running `npm run drizzle:studio`,",
+        "",
+        " Check out the Drizzle docs for more info:",
+        "   - https://orm.drizzle.team/docs/overview"
+      ]
+    }
+  },
+  "devDependencies": {
+    "drizzle-kit": "^0.20.14"
+  },
+  "dependencies": {
+    "drizzle-orm": "^0.29.1",
+    "@neondatabase/serverless": "^0.8.1",
+    "pg": "^8.11.3"
```

Comment:
Is it not possible to install a lighter DB by default? Perhaps there's something builtin without any deps?

Reply:
If not, how about sqlite? That would allow it to work out-of-the-box

Reply:
i had thought drizzle doesnt support schemas with sqlite as their docs says but it seems it does, just not multiple schemas? im still confused by that, checking with their support, but i did switch to better-sqlite3 as it is the documented one in their docs